eventually/authentication/models.py

```diff
@@ -0,0 +1,161 @@
+from django.db import models
+
+# Create your models here.
+
+from django.db import models
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.utils.dateformat import format
+from  datetime import datetime
+
+class CustomUser(AbstractBaseUser):
+    """
+        This class represents a basic user, containing only the essential information about the user.\n
+        1. first, last and middle names: all Char with maximum length of 20 chars.
+        2. email: email field of max length 20 chars
+        3. password: charfield of max length 128 symbols
+        4. updated_at & created_at fields: DateTimeField.
+           created_at can't be changed once created
+    """
+
+    #id - already exists
+    first_name = models.CharField(('first name'), max_length = 20)
+    middle_name = models.CharField(('middle name'), max_length=20)
+    last_name = models.CharField(('second name'), max_length=20)
+    email = models.EmailField(('email address'), max_length = 40, unique = True)
+    password = models.CharField(('password'), max_length = 128)
+    updated_at = models.DateTimeField(auto_now_add = True)
```

Comment:
updated_at should be auto_now cause auto_now is changing every time u use method save()

Reply:
Changed from auto_now_add to auto_now